charts/nri-bundle/requirements.lock

```diff
@@ -4,7 +4,7 @@ dependencies:
   version: 2.0.1
 - name: nri-prometheus
   repository: file://../nri-prometheus
-  version: 1.6.0
+  version: 1.6.2
```

Comment:
I am not entirely sure why `helm dependency update` changed this, the `requirements.yaml` file still points to 1.6.0.

Reply:
The local version you had is 1.6.2 if you are in `origin/main` 

https://github.com/newrelic/helm-charts/releases/tag/nri-prometheus-1.6.2

Even if 
```
  - name: nri-prometheus
    repository: file://../nri-prometheus
    condition: prometheus.enabled
    version: 1.6.0
```
is referenced, since you are checking it locally it gets the 1.6.2 and in the lock the inconcruence is shown

Reply:
I have never understood why we check for the chart locally and not in the helm repo.

Reply:
Interesting, I hadn't notice the `file://` URI. Should I uncommit the change just in case then?

Reply:
I would bump nri-prometheus as well and rerun helm dependency update.

Now I believe that we are packaging the 1.6.2 version calling it 1.6.0